src/cmdlib.sh

```diff
@@ -1,5 +1,13 @@
 # Shared shell script library
 
+# Global variables
+export workdir=$(pwd)
+export configdir=${workdir}/src/config
+export manifest=${configdir}/manifest.yaml
+export superminpreparedir="${workdir}/tmp/supermin-prepare.d"
+export superminbuilddir="${workdir}/tmp/supermin-build.d"
+export cachesimg="${workdir}/caches.qcow2"
```

Comment:
Some of this could be split into a prep commit.

Reply:
yep. i want to do that and also was thinking about making global variables all caps.